config/main.py

```diff
@@ -782,6 +792,64 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN', vlan_name, vlan)
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
+@vlan.group('dhcp_relay')
+@click.pass_context
+def vlan_dhcp_relay(ctx):
+    pass
+
+@vlan_dhcp_relay.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('dhcp_relay_ip', metavar='<dhcp_relay_ip>', required=True)
+@click.pass_context
+def add_vlan_dhcp_relay(ctx, vid, dhcp_relay_ip):
+    """ Add a DHCP relay address to the vlan """
+    if not is_ipaddress(dhcp_relay_ip):
+        ctx.fail('Invalid IP address')
+    db = ctx.obj['db']
+    vlan_name = 'Vlan{}'.format(vid)
+    vlan = db.get_entry('VLAN', vlan_name)
+
+    if len(vlan) == 0:
+        ctx.fail("{} doesn't exist".format(vlan_name))
+    dhcp_relay_dests = vlan.get('dhcp_servers', [])
+    if dhcp_relay_ip in dhcp_relay_dests:
+        ctx.fail("DHCP relay address {} is already a member of {}".format(dhcp_relay_ip, vlan_name))
+    else:
+        click.echo("Added DHCP relay address {} to {}".format(dhcp_relay_ip, vlan_name))
```

Comment:
Change to `click.echo("Added DHCP relay destination address {} to {}".format(dhcp_relay_ip, vlan_name))`

Also move below the `db.set_entry()` line so that past tense is appropriate.